jooq/src/main/resources/META-INF/native-image/io.micronaut.sql/jooq-graal/reflect-config.json

```diff
@@ -0,0 +1,8 @@
+[
```

Comment:
We need this because of JOOQ upgrade (from 3.17.7 to 3.18.2). The file is needed just until PR for `org.jooq:jooq:3.18.2` metadata gets approved and new version of metadata released.